_doc-build-windows/action.yml

```
@@ -278,23 +292,27 @@ runs:
       run: |
         python -m pip install pipx
         python -m pipx install poetry
-        python -m venv .venv
       env:
         PIPX_BIN_DIR: ${{ runner.temp }}/pipx/bin
         PIPX_HOME : ${{ runner.temp }}/pipx/home
 
     - name: "Create a virtual environment"
-      if: steps.determine-environment-variables.outputs.BUILD_BACKEND == 'pip'
       shell: powershell
       run: python -m venv .venv
 
     # ------------------------------------------------------------------------
 
     - name: "Update pip"
       shell: powershell
+      env:
+        USE_UV: ${{ inputs.use-uv }}
       run: |
         .venv\Scripts\Activate.ps1
-        python -m pip install -U pip
+        if ($env:USE_UV -eq "true") {
+            uv pip install -U pip
+        } else {
+            python -m pip install -U pip
+        }
```

```suggestion
        if ($env:USE_UV -eq "true") {
          uv pip install -U pip
        } else {
          python -m pip install -U pip
        }
```
To match files formatting

---

Thanks for catching this and subsequent ones @SMoraisAnsys!